pygsheets/cell.py

```diff
@@ -171,6 +173,21 @@ def set_text_format(self, attribute, value):
         self.update()
         return self
 
+    def set_text_rotation(self, attribute="angle", value=0):
+        """
+        set the text rotation
+        :param attribute: "angle"
+        :param value: corresponding value for the attribute
+        :return: :class: Cell
+        """
+        if attribute != "angle":
```

Comment:
I think that is because you were trying to set both, it can only be one of them.